src/com/kainos/buildasub/oop/BuildASub.java

```diff
@@ -0,0 +1,152 @@
+package com.kainos.buildasub.oop;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class BuildASub {
+
+    public static String chooseBread() {
+        String breadType;
+
+        System.out.println("Choose a bread:\n" +
+                "\t1. Flour\n" +
+                "\t2. Wholemeal");
+        Scanner scan = new Scanner(System.in);
+        int breadOption = scan.nextInt();
+
+
+
+        switch (breadOption) {
+            case 1: breadType = "Flour";
+                break;
+            case 2: breadType = "Wholemeal";
+                break;
+            default: breadType = "Flour";
+        }
+
+        return breadType;
+    }
+
+    public static String chooseMeat() throws Exception {
+        String meatType = "";
+
+        System.out.println("Choose a meat:\n" +
+                "\t1. Ham\n" +
+                "\t2. Bacon\n" +
+                "\t3. Goat");
+        Scanner scan = new Scanner(System.in);
+        int meatOption = scan.nextInt();
+
+
+
+        switch (meatOption) {
+            case 1: meatType = "Ham";
+                break;
+            case 2: meatType = "Bacon";
+                break;
+            case 3: meatType = "Goat";
+        }
+
+        if (!meatType.isEmpty()) {
+            return meatType;
+        }
+        else {
+            throw new Exception("Meat not set!");
+        }
+    }
+
+    public static List<String> ChooseCheese(){
+        List<String> SelectedCheese = new ArrayList<String>();
+        Boolean ContinueEnteringCheese = true;
+
+            System.out.println("Do you want cheese (Y/N)?");
+            Scanner scan = new Scanner(System.in);
+            String enteredOption = scan.nextLine();
+            if(enteredOption.toUpperCase().equals("Y")){
```

Comment:
While this solves the problem and is great I think it would be easier to maintain and more readable if you used a foreach statement along with a static array of cheese types:
```String[] cheeses = new String[]{"Regaular"}```

```        for (String cheese : cheeses) {
        }```